azure-pipelines.yml

```diff
@@ -133,9 +133,11 @@ jobs:
         --force
     displayName: 'Install toolchain'
   - bash: |
+      export BUILD_ROOT="$(Build.ArtifactStagingDirectory)"
+      . ci/build_consts.sh
       ./meson_init.sh -f
-      ninja -C build-verilator all
-      ninja -C build-fpga all
+      ninja -C "$(sw_obj_dir sim)" all
```

Comment:
Considering DV is also "sim", can we switch back to referring to "verilator" as "verilator" instead?

Reply:
@sriyerg aren't you using also binaries compiled with SIM=1 for DV runs that need software? Or are there special compile flags that you use?

Reply:
We are not setting SIM=1 when generating the binary. AFAIK, SIM=1 causes -DSIMULATION preprocessor macro to be set, which is used to fix the UART to a certain baud rate. We have no such limitation in DV. (I would like to change this as well to -DVERILATOR_SIM). 

I am sure the differences between DV and Verilator based sims will continue to grow, so it would perhaps be best to keep the identifiers separate. For DV we are using -DDV_SIM.

Reply:
Yes, it's the baud rate and the clock frequency to get the timing loops closer to wall-clock time in simulation. 

"verilator" itself is a very overloaded term already, so sim-verilator and sim-dv or something along those lines could be a suitable way out.